web/client/codechecker_client/cmd/store.py

```diff
@@ -143,6 +143,23 @@ def add_arguments_to_parser(parser):
                              "If multiple prefix is given, the longest match "
                              "will be removed.")
 
+    parser.add_argument('--config',
+                        dest='config_file',
+                        required=False,
+                        help="Allow the configuration from an explicit JSON "
+                             "based configuration file. The value of the "
+                             "'store' key in the config file will be "
+                             "emplaced as command line arguments. The format "
+                             "of configuration file is: "
+                             "{"
+                             "  \"enabled\": true,"
```

Comment:
To me this enabled value is a bit confusing, if I want to use a config file I set it in the command line, if not I do not set it.
But why would I set it in the command line and than disable it in the config file?
@bruntib @dkrupp what do you think?

Reply:
The [`.codechecker.passwords.json`](https://github.com/Ericsson/codechecker/blob/master/docs/web/authentication.md#preconfigured-credentials-) and the [`server_config.json`](https://github.com/Ericsson/codechecker/blob/master/web/server/config/server_config.json) look the same: both have an option to turn off or on the feature. I wanted to be more consistent with these features. By the way the analyzer --config option works the same so if we would like to change this behaviour we can do it in another pull request. Maybe we can create an issue for this.

Reply:
I've created a separate issue to discuss it #2654